### ISSUE TITLE
404 error with auto shipit even with GHE god mode token 😖

### ISSUE THREAD

**Issue by RichieRunner** [intuit/auto#1340]
I'm trying to use this library with a **GHE account.**
Labels have already been set up, with githubApi url endpoint correctly declared.

`auto info` confirms that token and initial setup has been done correctly:
![image](https://user-images.githubusercontent.com/20289967/86403247-2f069980-bc62-11ea-87a0-f0f7ac07e993.png)

However, running `auto release` and `auto shipit` still gives 404 error:

```
This usually because the GitHub token you're using doesn't have the correct permissions to the repo.

The token used with auto must have at least "write" permission to your repo
```

However, there isn't any more access i can grant to this token...
![image](https://user-images.githubusercontent.com/20289967/86403424-8ad12280-bc62-11ea-948f-e75a15f50365.png)

This is happening locally as well as in CI (AWS Codebuild)

**Comment by hipstersmoothie** [comment]
Can you run with `-v` using the [latest version](https://github.com/intuit/auto/releases/tag/v9.40.6)? I want to see the full error

**Comment by RichieRunner** [comment]
Environment Information:
--
468 |  
469 | "auto" version: v9.40.6
470 | "git"  version: v2.27.0
471 | "node" version: v10.21.0GHE version:    v2.20.6



![image](https://user-images.githubusercontent.com/20289967/86501048-5d5fa400-bd4a-11ea-923e-66cbf396f661.png)

**Comment by RichieRunner** [comment]
I am not declaring a githubGraphQLApi value in my `.autorc` file, so this was what was derived. The graphqlBaseUrl seems to be same as my githubAPI url, ,not sure if that makes a difference or not.

![image](https://user-images.githubusercontent.com/20289967/86501118-deb73680-bd4a-11ea-98e9-f2af274fc6ee.png)

**Comment by hipstersmoothie** [comment]
I think this is the problem. When I use `auto` on an enterprise project I always set the following

```json
{
  "githubApi": "https://github.INTERNAL.com/api/v3",
  "githubGraphqlApi": "https://github.INTERNAL.com/api"
}
```

We might be able to automate this more. Can you check if changing this setting helps at all?

**Comment by hipstersmoothie** [comment]
Also you're token doesn't need that many permission. just the "repo" is fine

**Comment by hipstersmoothie** [comment]
Re-opening until I know the issue is resolved. The newest version should build the graphql url for enterprise github without any extra settings now. Would appreciate if you would test and report back if it works!

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v9.41.1`](https://github.com/intuit/auto/releases/tag/v9.41.1) :rocket:

**Comment by RichieRunner** [comment]
That fixed it; thanks for taking a look!
